src/main/java/org/junitpioneer/jupiter/params/ByteOrder.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2016-2022 the original author or authors.
+ *
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License v2.0 which
+ * accompanies this distribution and is available at
+ *
+ * http://www.eclipse.org/legal/epl-v20.html
+ */
+
+package org.junitpioneer.jupiter.params;
```

Comment:
Still taking inspiration from `JavaTimeConversionPattern`, JUnit uses a dedicated package for converters:

```suggestion
package org.junitpioneer.jupiter.params.converter;
```